internal/handlers/pg/msg_dbstats.go

```diff
@@ -41,13 +41,16 @@ func (h *Handler) MsgDBStats(ctx context.Context, msg *wire.OpMsg) (*wire.OpMsg,
 		return nil, err
 	}
 
-	m := document.Map()
-	scale, ok := m["scale"].(float64)
```

Comment:
I tried something like `db.runCommand( {"dbStats": 1, "scale": 1024}`, but it didn't work (in fact, it considered `scale = 1`), so I mark `scale` as `UnimplementedNonDefault`.